apps/rewards/src/invite/inviteReward.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm'
+
+export enum RewardStatus {
```

Comment:
We'll probably need a state for "Success" and "DeadLettered" but we can add it once we hook up the tx submission pipeline.

Reply:
`Success` would be the `Sent` one right?

Reply:
I was thinking `Sent` implies that the tx was accepted into the mem-pool and `Success` is that the tx was finalized in a block.

Reply:
Made this distinction in this PR: https://github.com/celo-org/komenci/pull/179